presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -55,6 +55,9 @@ statement
         (GRANT OPTION FOR)?
         (privilege (',' privilege)* | ALL PRIVILEGES)
         ON TABLE? qualifiedName FROM grantee=identifier                #revoke
+    | SHOW GRANT
+        (USER identifier)?
```

Comment:
I could do that. I kept it to distinguish between this case and the case of `ROLE identifier` which I plan to add later.